plugins/modules/network_settings_intent.py

```diff
@@ -1949,6 +1978,104 @@ def get_diff_deleted(self, config):
 
         return self
 
+    def verify_diff_merged(self, config):
+        """
+        Validating the DNAC configuration with the playbook details
+        when state is merged (Create/Update).
+
+        Parameters:
+            config (dict) - Playbook details containing Global Pool,
+            Reserved Pool, and Network Management configuration.
+
+        Returns:
+            self
+        """
+
+        self.get_have(config)
+        self.log(str(self.have))
+        self.log(str(self.want))
+        if config.get("global_pool_details") is not None:
+            obj_params = self.get_obj_params("GlobalPool")
```

Comment:
Can we change it into
global_obj_params or self.global_obj_params and reuse in verify?